### DISCUSSION TITLE
Running on Pi3B with 3,5" RPi Display

### DISCUSSION THREAD

**Discussion by cniweb** [taoteh1221/Slideshow_Crypto_Ticker#26] [Show and tell]
I also wanted to show you how a Raspberry Pi 3B looks like with a 3.5" RPi display and a suitable housing:
![grafik](https://user-images.githubusercontent.com/2334906/118412810-54469f80-b69c-11eb-80e1-1118d4e241b5.png)

**Comment by taoteh1221** [comment]
Looks good, I have a case like that as well!
